src/components/interactive-image/interactive-image.tsx

```diff
@@ -0,0 +1,222 @@
+import { Component, Element, Prop, h, State, Watch } from "@stencil/core";
+import { Component as GxComponent } from "../common/interfaces";
+
+@Component({
+  shadow: false,
+  styleUrl: "interactive-image.scss",
+  tag: "gx-interactive-image"
+})
+export class InteractiveImage implements GxComponent {
+  @Element() element: HTMLGxInteractiveImageElement;
+
+  /**
+   * If this property is true, the user can copy the image to the clipboard (Only for iOS, see image below).
+   */
+  @Prop() enableCopyToClipboard: false;
```

Comment:
We shouldn't add this property if we won't be doing anything with it in this PR.

Reply:
Yeap, it was removed now. Look at commit > b764a48